package.json

```diff
@@ -108,6 +108,7 @@
     "oauth-1.0a": "^2.2.6",
     "object-hash": "^3.0.0",
     "parse-static-imports": "^1.1.0",
+    "piscina": "^4.3.0",
```

Comment:
I see that they published `piscina: 5.0.0`. Can you give it a try @satishrudderstack ?

Reply:
Upgraded 5.0.0